gbpservice/neutron/nsf/modules/config_agent.py

```diff
@@ -0,0 +1,75 @@
+import os
```

Comment:
Discussed with Ahmed, All modules must be in nfp/modules. Each module implementation then can be in a separate folder. Ahmed spoke to jagadish and configurator is also following the same.